cyences_app_for_splunk/default/macros.conf

```diff
@@ -94,6 +94,22 @@ args = ip_field
 definition = eval is_source_internal = if(cidrmatch("10.0.0.0/8",$ip_field$) OR cidrmatch("172.16.0.0/12",$ip_field$) OR cidrmatch("192.168.0.0/16",$ip_field$), "Yes", "No") 
 iseval = 0
 
+[cs_is_private_ip(1)]
```

Comment:
we have similar macro for this defination: **cs_identify_private_ips**

Reply:
But new macro name seems more appropreaite.

Existing macro might be in use already somewhere at customer filters, hence I have not renamed and created new one.

Reply:
This macro is not available to modify from the cyences UI as this is the common definition. You can remove cs_identify_private_ips and replace the name cs_is_private_ip everywhere.